plugin.video.vimeo/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.vimeo" name="Vimeo" version="5.0.0" provider-name="jaylinski">
```

Comment:
Are u using any of the old code from bromix? If so please add bromix also as an author

Reply:
Nope, I only copied the Vimeo icon and fanart. The code is new.